msal/tests/dev apps/XForms/XForms/AcquirePage.xaml.cs

```diff
@@ -160,7 +160,7 @@ private string GetSelectedUserId()
             }
             catch (MsalException exception)
             {
-                acquireResponseLabel.Text = "MsalException - " + exception.Message;
+                acquireResponseLabel.Text = String.Format("MsalException -\nError Code: {0}\nMessage: {1}", exception.ErrorCode, exception.Message);
```

Comment:
The Xamarin app was just showing `MsalServiceException`, so now it returns a little more info. not all the exceptions have a message.